scanner/cmd/scanner/main.go

```diff
@@ -103,19 +112,34 @@ func createGRPCService(backends *Backends) (grpc.API, error) {
 	if err != nil {
 		return nil, fmt.Errorf("identity extractor: %w", err)
 	}
-	// Create gRPC API service.
+	// Create gRPC API service and debug routes.
+	var customRoutes []routes.CustomRoute
```

Comment:
nit: `customRoutes := make([]routes.CustomRoute, 0, len(routes.DebugRoutes))`

Reply:
I am OK performing a couple of additional allocations once in the lifetime of Scanner.

Reply:
We tend to try to follow the convention of: if you know how large the slice will be, then pre-allocate all the space.

I won't block the PR over this

Reply:
6c4f286228..46b9e7723d